src/global-messages/types.ts

```diff
@@ -21,6 +21,8 @@ export type GlobalMessageRaw = {
   appPlatforms: AppPlatformType[];
   appVersionMin: string;
   appVersionMax: string;
+  startDate?: number;
+  endDate?: number;
```

Comment:
For å gjøre dette helt korrekt så bør det være type `Timestamp` i `GlobalMessageRaw`, mens i `GlobalMessageType` så bør det være `number`. I fix slik at det blir merget nå før påske.